sbt-plugin/src/sbt-test/sbt-akkaserverless/basic/build.sbt

```diff
@@ -1,9 +1,3 @@
 scalaVersion := "2.13.6"
 
 enablePlugins(AkkaserverlessPlugin)
-
-val AkkaServerlessSdkVersion = System.getProperty("plugin.version", "0.7.2")
-libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "3.2.7" % Test,
```

Comment:
ScalaTest must remain here, if you agree with https://github.com/lightbend/akkaserverless-java-sdk/pull/508#discussion_r717260609